lib/routes/content.js

```diff
@@ -0,0 +1,160 @@
+  return new Promise(resolve => {
+    const app = application;
+    const types = app.get('content-types');
+
+    /*
+     * Content Home Page
+     */
+    app.get(`/${config.content.base}`, (req, res) => {
+      res.render('content/home', {
+        content: {
+          home: config.content.home,
+          base: config.content.base,
+          types,
+        },
+      });
+    });
+
+    /*
+     * Individual Content Type Landing Page
+     */
+    app.get(`/${config.content.base}/:type`, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+
+      if (type === false) {
+        _.set(req.session, '404', {
+          message: config.content.messages.missing.replace('%type', req.params.type),
+          safe: `/${config.content.base}`,
+        });
+
+        return next();
+      }
+
+      res.render('content/landing', {
+        content: {
+          base: req.url,
+          actions: config.content.actions,
+          type,
+        },
+      });
+
+      return true;
+    });
+
+    /*
+     * Individual Content Type Add Page
+     */
+    app.get(`/${config.content.base}/:type/${config.content.actions.add}`, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+      const errors = _.get(req.session, 'form.content.add.errors', {});
+      let values = _.get(req.session, 'form.content.add.content', {});
+
+      _.unset(req.session, 'form.content.add');
+
+      values = utils.config(values);
+
+      if (type === false) {
+        _.set(req.session, '404', {
+          message: config.content.messages.missing.replace('%type', req.params.type),
+          safe: `/${config.content.base}`,
+        });
+
+        return next();
+      }
+
+      content.only(req.params.type, values, [type]).then(merged => {
+        return content.form(merged, errors).then(form => {
+          res.render('content/add', {
+            form,
+            action: req.url.replace(config.content.actions.add, config.content.actions.save),
+            type,
+          });
+        });
+      }).catch(e => {
+        next(e);
+      });
+
+      return true;
+    });
+
+    /*
+     *  Post to Content Type
+     */
+    app.post(`/${config.content.base}/:type/${config.content.actions.save}`, multipartMiddleware, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+
+      if (type === false) {
+        _.set(req.session, '404', {
+          message: config.content.messages.missing.replace('%type', req.params.type),
+          safe: `/${config.content.base}`,
+        });
+
+        return next();
+      }
+
+      // Validation
+      const validated = content.form.validate(req.body, type);
+
+      // utils.log(validated);
+
+      if (validated === true) {
+        // Sunrise/Sunset
+        const sunrise = utils.time.iso(req.body['sunrise-date'], req.body['sunrise-time'], 'EDT');
```

Comment:
Will need to do that, didn't want to spend cycles updating the universal input stuff to account for timezones. Let's capture as a story and we can update